docs/examples/repositories_using_flax.rst

```diff
@@ -0,0 +1,43 @@
+
+***********************
+Repositories Using Flax
+***********************
```

Comment:
what's the relation of this & the "Ecosystem" on the main page?

Reply:
They turned up to be very similar, this section has more info however.

Reply:
I don't have strong opinions here. We could remove it if there is interest in doing so.

Reply:
I think it's fine having both. But in that case we should list *every* project from the main index page (e.g. dalle mini and palm are missing), and provide some additional information for each of them. But we don't need every example from here to be on the main landing page (that list should remain short and engaging).

Reply:
Reflecting on this a bit, some projects like Imagen and PaLM don't have Open Source repos so it doesn't make sense to include them in this section. I did add DALLE Mini.

Reply:
ah right.